### PR TITLE
Feature/90/sandbox form multiselect search

### PR DESCRIPTION
**Pull request by Dmitruk89**
Снимок экрана 2021-11-09 в 13 17 57
1. replace standard multi-select fields of create-sandbox page with searchable multi-select
2.  fix search filter in multi-select.

### CODE REVIEW

**Review by Nicolai-boiko — APPROVED** [review]

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -9,8 +9,8 @@ import { SandboxListPageComponent } from './pages/sandbox-list-page/sandbox-list
```
Comment (Nicolai-boiko) [inline src/app/app-routing.module.ts:9]: delete import { CandidateListComponent } from './components/candidate/candidate-list/candidate-list.component';

**Review by Nicolai-boiko — CHANGES_REQUESTED** [review]

**Review by Nicolai-boiko — APPROVED** [review]

**Comment by PaulRomanov** [comment]
looks without errors

**Review by PaulRomanov — APPROVED** [review]

File: src/app/components/candidate/multiselect-search/multiselect-search.component.scss [hunk 2]
```diff
@@ -4,13 +4,14 @@
...
+
   .select-container{
     margin-top:10px;
   }
 
   .selected {
-    width: 500px;
-    position: absolute;
+    margin-left: 20px;
+    color: rgba(0,0,0,0.6);
```
Comment (vstrabko) [inline src/app/components/candidate/multiselect-search/multiselect-search.component.scss:14]: color should be in colors scss file :)

File: src/app/pages/create-sandbox-page/create-sandbox-page.component.scss [hunk 3]
```diff
@@ -32,3 +32,13 @@ $width-content: calc(100% - 234px);
     max-height: 250px;
   }
 }
+
+.multi_flex {
+  display: flex;
+  align-items: baseline;
+  background-color: rgba(0,0,0,.04);
```
Comment (vstrabko) [inline src/app/pages/create-sandbox-page/create-sandbox-page.component.scss:39]: Same here

**Review by vstrabko — APPROVED** [review]
Looks good. Just 2 small comments

File: src/app/components/candidate/multiselect-search/multiselect-search.component.scss (hunk 2 above)
Comment (Dmitruk89) [reply src/app/components/candidate/multiselect-search/multiselect-search.component.scss]: fixed

**Pull request merged**